mimic/model/clb_objects.py

```diff
@@ -38,6 +42,28 @@ def __init__(self):
         self.lbs = {}
         self.meta = {}
 
+    def __len__(self):
```

Comment:
All good points; my goal was to more clearly express intent in the REST and test modules.  However, you're right in that they can mistakenly mislead a coder into thinking that the regional classes are sufficiently container-like that they can be used interchangably with them.

I'll rename the methods that I use in my code (and eliminate those required to just implement the minimum interface requirements for containership).